docs/api/index.md

```diff
@@ -2917,7 +3008,45 @@ IntersectionObserver === undefined
 
 - **Type**: `(path: string) => void`
 
-  Removes module from mocked registry. All subsequent calls to import will return original module even if it was mocked.
+  Removes module from mocked registry. All calls to import will return original module even if it was mocked before. This call is hoisted (moved) to top of the file, so it will only unmock modules that were defined in `setupFiles`, for exampl.
```

Comment:
```suggestion
  Removes module from the mocked registry. All calls to import will return the original module even if it was mocked before. This call is hoisted (moved) to the top of the file, so it will only unmock modules that were defined in `setupFiles`, for example.
```